scanapi/scan.py

```diff
@@ -33,7 +32,7 @@ def scan():
         error_message = f"API spec file is empty. {str(e)}"
         logger.error(error_message)
         raise SystemExit(ExitCode.USAGE_ERROR)
-    except (yaml.YAMLError, FileFormatNotSupportedError) as e:
+    except yaml.YAMLError as e:
         logger.error(e)
```

Comment:
Maybe
```suggestion
    except yaml.YAMLError as excep:
        logger.error(excep)
```
instead?

Reply:
`e` is a pretty common name so this one I'm unsure about.